### PR TITLE
Add user-defined runtime prompt in app config

### PR DESCRIPTION
**Pull request by sejust**

### CODE REVIEW

**Comment by skaldesh** [comment]
Interesting! I wll to take a closer look at this

File: sample/simple/cmd/app.go [hunk 1]
```diff
@@ -26,15 +26,39 @@ package cmd
...
 
 var App = grumble.New(&grumble.Config{
 	Name:        "foo",
 	Description: "An awesome foo bar",
+	PromptRuntime: func() string {
+		username := "NO-USER"
+		hostname := "NO-HOSTNAME"
+		user, err := user.Current()
+		if err == nil {
+			username = user.Name
```
Comment (skaldesh) [inline sample/simple/cmd/app.go:46]: On my system, `Name` is empty. Using `Username` yields the correct name.

**Review by skaldesh — CHANGES_REQUESTED** [review]

**Comment by skaldesh** [comment]
@r0l1 What do you think of this?

File: sample/simple/cmd/app.go (hunk 1 above)
Comment (sejust) [reply sample/simple/cmd/app.go:46]: This runtime function just implement as sample, i will fix it later.

**Comment by r0l1** [comment]
Pull request was reverted, because there is a SetPrompt command.

**Pull request merged**